src/django_idom/components.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from django.contrib.staticfiles.finders import find
+from idom import component, html
+
+from django_idom.config import IDOM_CACHE
+
+
+@component
+def DjangoCSS(static_path: str):
+    """Fetches a CSS static file for use within IDOM. This allows for deferred CSS loading.
+
+    Args:
+        static_path: The path to the static file. This path is identical to what you would
+        use on a `static` template tag.
+    """
+    return html.style(_cached_static_contents(static_path))
+
+
+@component
+def DjangoJS(static_path: str):
+    """Fetches a JS static file for use within IDOM. This allows for deferred JS loading.
+
+    Args:
+        static_path: The path to the static file. This path is identical to what you would
+        use on a `static` template tag.
+    """
+    return html.script(_cached_static_contents(static_path))
+
+
+def _cached_static_contents(static_path: str):
+    # Try to find the file within Django's static files
+    abs_path = find(static_path)
+    if not abs_path:
+        raise FileNotFoundError(
+            f"Could not find static file {static_path} within Django's static files."
+        )
+
+    # Fetch the file from cache, if available
+    last_modified_time = os.stat(abs_path).st_mtime
+    cache_key = f"django_idom:static_contents:{static_path}"
+    file_contents = IDOM_CACHE.get(cache_key, version=last_modified_time)
```

Comment:
Could you add a comment about the fact that this is preferable to `use_memo` because the cache works across-processes?